docs/app/Examples/views/Advertisement/Types/AdvertisementExampleStandard.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+import { Advertisement } from 'semantic-ui-react'
+
+export default class AdvertisementExampleStandard extends Component {
+  componentDidMount() {
+    (window.adsbygoogle = window.adsbygoogle || []).push({})
```

Comment:
Example components are re-mounted on every change in the editor, this will be called several times.

Also, we should check if the `adsbygoogle` object is loaded, then push an ad after it is loaded.  Let's move this to a script tag in `index.ejs` immediately after the google ads script is loaded.